### PR TITLE
Debug server side components with Chrome Developer Tools (DevTools)

### PR DESCRIPTION
**Pull request by dennisat**
usage
```
gulp server-devtools
gulp server-devtools-startup-breakpoint
gulp server-build-devtools
gulp server-build-devtools-startup-breakpoint
```

### CODE REVIEW

**Comment by dennisat** [comment]
I doesn't look my changes break the test. Any suggestions?

**Comment by dennisat** [comment]
@jchip do you have any idea why the build fails?

**Comment by jchip** [comment]
there's a test that timing out randomly.  

thanks for the PR.  the new task names look a bit long.

anyways, merging for now.  I might shorten them later.

File: packages/electrode-archetype-react-app/arch-gulpfile.js [hunk 1]
```diff
@@ -475,6 +475,35 @@ Individual .babelrc files were generated for you in src/client and src/server
...
+    "server-devtools": {
+      desc: "Debug the built app (in dist/)",
+      task: () => {
+        console.log("server-devtools HELP: To terminate this server Control-Break this AND also close the devTools!");
+        AppMode.setEnv(AppMode.lib.dir);
+        return exec(`node --inspect ${Path.join(AppMode.lib.server, "index.js")}`);
+      }
+    },
+
+    "server-devtools-startup-breakpoint": {
```
Comment (jchip) [inline packages/electrode-archetype-react-app/arch-gulpfile.js]: `server-start-debug`?

File: packages/electrode-archetype-react-app/arch-gulpfile.js [hunk 2]
```diff
@@ -475,6 +475,35 @@ Individual .babelrc files were generated for you in src/client and src/server
       return exec(`node debug ${Path.join(AppMode.lib.server, "index.js")}`);
     },
 
+    "server-build-devtools": {
+      desc: "Build and debug with devTools",
+      task: ["build", "server-devtools"]
+    },
+
+    "server-build-devtools-startup-breakpoint": {
```
Comment (jchip) [inline packages/electrode-archetype-react-app/arch-gulpfile.js]: `server-build-start-debug`?

**Pull request merged**